pkg/controller/clusterissuers/checks.go

```diff
@@ -60,11 +60,11 @@ func (c *controller) issuersForSecret(secret *corev1.Secret) ([]*v1.ClusterIssue
 					affected = append(affected, iss)
 					continue
 				}
-			}
-			if iss.Spec.Venafi.TPP.CABundleSecretRef != nil {
-				if iss.Spec.Venafi.TPP.CABundleSecretRef.Name == secret.Name {
-					affected = append(affected, iss)
-					continue
+				if iss.Spec.Venafi.TPP.CABundleSecretRef != nil {
+					if iss.Spec.Venafi.TPP.CABundleSecretRef.Name == secret.Name {
+						affected = append(affected, iss)
+						continue
+					}
 				}
```

Comment:
This imbrication of `case` and `if` is hard to follow... what about flattening this?

```go
		case iss.Spec.Venafi != nil &&
			iss.Spec.Venafi.TPP != nil &&
			iss.Spec.Venafi.TPP.CredentialsRef.Name == secret.Name:
					affected = append(affected, iss)
					continue
```

The repetition isn't a problem IMO.

Reply:
I think a refactor would be useful here, but should be in a separate PR. This is just a minimal change to fix the issue.

Reply:
Excellent point.